packages/lib-subject-viewers/.babelrc

```diff
@@ -0,0 +1,48 @@
+{
+  "plugins": [
+    "babel-plugin-styled-components",
+    [
+      "transform-imports",
+      {
+        "lodash": {
+          "transform": "lodash/${member}"
+        }
```

Comment:
Is lodash used in the viewer component? If not please remove. This was copy pasted from another library.